Geometry/VeryForwardData/data/RP_Detectors_Assembly/RP_Detectors_Assembly_001.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
are these files auto-generated?
If there is a clear prescription how elements are placed, it would be better to define all the copies algorithmically.